### PR TITLE
Custom fields in config

### PR DESCRIPTION
**Pull request by manwithtwowatches**
Can now display custom field data on cards. If the custom_field property on the column is true then it will look for that property on the object given and display the data if present.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,12 +126,22 @@ group2:
...
+          has_label: true
+          font_size: 15
+          minimum_lines: 1
 </pre>
 
 Any number of rows can be defined. The font_size and minimum_lines properties define how much height on the card is given to the row. The plugin will attempt to assign enough space to each of the rows, however space will be assigned based on the priorities of the the rows, with rows with lower priority (higher numbers) being reduced and removed first if there is not enough for all the data. The row height can be forced by giving a value, in pixels, for the row height property. This will override the assigned row height.
 
 The name of the column informs the plugin which data should be read from the model (status, due_date, id, etc.). There can be any number of columns per row. Columns are given an equal share of the row width unless a specific width % is given. If there is more text in the column than can fit into its assinged space on the card then the text will be truncated.
 
+A column can also contain data from a custom field. The column must have the property custom_field set to true in order for it to appear.
```
Comment (ulferts) [inline README.md]: Why do you need the user to specify it as a custom field?

You could alternately just look all all configured custom fields and see if one fits.

Comment (manwithtwowatches) [inline README.md]: @ulferts A moment of mental weakness:> changing now.

**Comment by ulferts** [comment]
I believe you should change the Readme to reflect the change.

**Comment by ulferts** [comment]
Things I noticed:
1) it is possible to specify attributes that do not exist(e.g. "blubs"). The attribute name will be rendered (at least if render_if_empty is set)
2) The name of the custom field is only checked in the current language. Custom fields can have multiple languages (well two) if the settings of the installation permits it. So if I have a custom field that is named "Field1" and "Feld1" and I specified "Field1" to be exported, it will not be exported if my locale is :de
3) The values seem only to be exported if the render_if_empty value is set.

**Comment by manwithtwowatches** [comment]
I have made changes for 1) and 3) but regarding 2)...

Correct me if i'm wrong but when you create a custom field with multiple languages it just creates multiple custom fields. I don't see any way of translating between the related custom fields because their names are not stored in the locale configs. I'll look again in the morning when i'm less exhausted though.

**Comment by ulferts** [comment]
Custom fields use globalize3 to allow setting names, available_values and default_values in multiple locales. So there is a custom_fields table and a custom_field_translations table. A custom_field can have multiple translations in the custom_field_translations table.

**Pull request merged**